### PR TITLE
Scanning screen

### LINKED ISSUE #24: Scanning screen
# Goal

We should have the UI in place for the "Scanning" screen which allows users to know that the app is currently scanning for beacons

# Plan

1. Use wireframe to implement initial UI
1. Link this screen to a "scanning" state which Conor can change using the loop
1. Potentially switch between screens when switching between states?

# Additional information

Refer to wireframes sent to Marion

### PR DESCRIPTION
**Pull request by Ashwin-MJ**
# Details

- Created a "Scanning" screen which displays that the app is currently looking for beacons
- Fixed navigation between screens, and removed the tab bar at the bottom (no longer needed)
- Modified header for screens (can update this easily in the future as desired)
- Fixed `.gitignore` to remove unnecessary Xcode files

# Screenshots/GIF (if appropriate)

Note that the button to switch screens is temporary, in the future navigation between screens will be based on beacon detection and the button will be removed

- iOS:
drawing

- Android (smaller device):
drawing

# Checklist

## Code
- [x] I have added some tests
- [x] I have run the tests locally to ensure they all pass

## Documentation
N/A

## Accessibility
- [x] Tested Accessibility on **Android**
  - [x] Any added fields can be read aloud by the screen reader
- [x] Tested Accessibility on **iOS**
  - [x] Any added fields can be read aloud by the screen reader

Closes #24

### CODE REVIEW

**Review by miklasr — DISMISSED** [review]

File: app/components/ScanningButton.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import styles from './styles/ScanningButton.style';
+
+import * as Speech from 'expo-speech';
+
+class ScanningButton extends Component {
+  render() {
+    return (
+      <Pressable
+        style={styles.buttonContainer}
+        android_ripple={{ color: '#fff' }}
```
Comment (C-Begley) [inline app/components/ScanningButton.tsx]: Can the ripple colour also be added to the style sheet?

Comment (Ashwin-MJ) [reply app/components/ScanningButton.tsx]: You can't move it to the style sheet, but I have moved it to the `DefaultColors.tsx` file so we can modify it easily in the future if we want to

**Review by C-Begley — APPROVED** [review]
I'm happy with the change, looks good to me

**Review by Andrew-Ritchie — APPROVED** [review]
Looks good to me! Nice work.

**Review by miklasr — APPROVED** [review]

**Pull request merged**